src/Listing/ScriptFinder.php

```diff
@@ -71,6 +71,19 @@ public function getAllScripts(): array
     }
 
     /**
+     * @param string $query
+     * @return array
+     */
+    public function findScriptsByName(string $query): array
+    {
+        $scripts = $this->getAllScripts();
+
+        return array_filter($scripts, function ($key) use ($query) {
```

Comment:
I like it. You could get one step further and use the levenshtein distance (http://php.net/manual/de/function.levenshtein.php) to account for small typos.